app/models/miq_queue.rb

```diff
@@ -541,6 +543,13 @@ def self.get_worker(task_id)
 
   private
 
+  def activate_miq_task(args)
+    MiqTask.update_status(miq_task_id, MiqTask::STATE_ACTIVE,  MiqTask::STATUS_OK, "Task starting") if miq_task
+    pars = args.first
```

Comment:
Prefer `params` or `parameters`...`pars` is not a common abbreviation, so it reads funny.

Reply:
ok